apps/web/pages/api/book/event.ts

```diff
@@ -761,29 +756,19 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         metadata.entryPoints = results[0].createdEvent?.entryPoints;
       }
       if (noEmail !== true) {
-        await sendScheduledEmails(
-          {
-            ...evt,
-            additionalInformation: metadata,
-            additionalNotes,
-            customInputs,
-          },
-          reqBody.recurringEventId ? (eventType.recurringEvent as RecurringEvent) : {}
-        );
+        await sendScheduledEmails({
+          ...evt,
+          additionalInformation: metadata,
+          additionalNotes,
+          customInputs,
+        });
       }
     }
   }
 
   if (eventType.requiresConfirmation && !rescheduleUid && noEmail !== true) {
-    await sendOrganizerRequestEmail(
-      { ...evt, additionalNotes },
-      reqBody.recurringEventId ? (eventType.recurringEvent as RecurringEvent) : {}
-    );
-    await sendAttendeeRequestEmail(
-      { ...evt, additionalNotes },
-      attendeesList[0],
-      reqBody.recurringEventId ? (eventType.recurringEvent as RecurringEvent) : {}
-    );
+    await sendOrganizerRequestEmail({ ...evt, additionalNotes });
+    await sendAttendeeRequestEmail({ ...evt, additionalNotes }, attendeesList[0]);
```

Comment:
do we want to allow the booker (attendee[0] to select a checkbox when booking to notify ALL attendees and not him? Since we ask for emails for all attendees too?